### DISCUSSION TITLE
gifts not showing under tree

### DISCUSSION THREAD

**Discussion by Cosmic-Incantations** [Monika-After-Story/MonikaModDev#8459] [QA - tech support + general] [answered]
I keep trying to put gifts under the tree, but Monika opens them as soon as I give them to her! I give her chocolates, and she eats them! dose the tree thing only work for spite packs? cause it worked for the teal and silver ribbon, but when I tried to give her the other stuff, she reacted to them instead of putting them under the tree! then I thought it was a thing that happened once, so I turned back the clock to before the tree was up then back again just to find that there was no gift pile at all and she dose not put gifts under the tree! did I break something?

**Comment by multimokia** [comment, marked as answer]
Stuff that is consumable -- chocolates, fudge, hotchocolate, coffee, etc. will all be opened directly as some don't make sense to be opened for christmas or some may want to give them at the current moment.

Also please restore a persistent backup. Turning back the clock has 100% messed with the event and all of the things you've done w/ Monika over the event have _kinda_ been reset.

**Comment by Cosmic-Incantations** [comment]
ok, thank you. I felt like I would corrupt the persistent files if I did this, but I did it any way! this will teach me to listen to my gut and not endanger Monika by being dumb. Good thing I backed up the persistent files on google drive, cause I did stuff like this in the past so I knew this would happen lol! have a merry kiss-a-miss!

**Comment by multimokia** [comment]
No problem. Wish you the same!
